package.json

```diff
@@ -20,7 +20,7 @@
 		"build-color-schemes-wpcom": "GULP_ENV=wpcom gulp sass:color-schemes",
 		"build-concurrently": "yarn install-if-deps-outdated && yarn clean && yarn concurrently 'yarn build-client' 'yarn build-php' 'yarn build-extensions' 'yarn build-search' 'yarn build-packages'",
 		"build-extensions": "webpack --config ./webpack.config.extensions.js",
-		"build-packages": "webpack --config ./webpack.config.packages.js",
+		"build-packages": "webpack --config ./webpack.config.packages.js && yarn --cwd ./packages/connection-ui build",
```

Comment:
Now that we have merged the reorg PR, can we go about building JS for this package the same way we do it for lazy images? I mean two things here:

1. Putting build files back into the package folder, and enqueuing them from there.
2. Using the same type of script that we use for lazy images in the package.json.

Reply:
We're doing it essentially the same way.
The only difference is that we use Gulp for this package, while Lazy Images setup is a bit simpler and doesn't need it.